packages/legacy/src/Providers/components/AddEditProviderModal/AddEditProviderModal.tsx

```diff
@@ -56,6 +57,7 @@ const PROVIDER_TYPE_OPTIONS = PROVIDER_TYPES.map((type) => ({
 
 const oVirtLabelPrefix = process.env.BRAND_TYPE === 'RedHat' ? 'RHV Manager' : 'oVirt Engine';
 const vmwareLabelPrefix = 'vCenter';
+const openStackLabelPrefix = process.env.BRAND_TYPE === 'RedHat' ? 'Red Hat OpenStack Identity' : 'OpenStack Identity';
```

Comment:
```suggestion
const openStackLabelPrefix = 'OpenStack Identity (Keystone)';
```

Reply:
This was for u/s - d/s distinguishing as well, but it's too long to add so ok